test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -47,7 +47,7 @@ public class IntegrationTestBase
         private readonly string XUnitTestAdapterRelativePath = @"xunit.runner.visualstudio\{0}\build\_common".Replace('\\', Path.DirectorySeparatorChar);
         private readonly string ChutzpahTestAdapterRelativePath = @"chutzpah\{0}\tools".Replace('\\', Path.DirectorySeparatorChar);
 
-        protected readonly bool IsWindows = System.Environment.OSVersion.Platform.ToString().StartsWith("Win");
+        protected static readonly bool IsWindows = System.Environment.OSVersion.Platform.ToString().StartsWith("Win");
```

Comment:
But what if the OS architecture changes mid-run? 😁

Reply:
Pls do not undermine my few certainties, anyway if it happens, I'll call for sure 😄